VMBackup/main/handle.py

```diff
@@ -603,20 +614,22 @@ def enable():
         para_parser = ParameterParser(protected_settings, public_settings, backup_logger)
 
         if(para_parser.taskId is not None and para_parser.taskId != ""):
+            eventlogger.update_properties(para_parser.taskId)
+            hutil.set_event_logger(eventlogger)
```

Comment:
this section of code is duplicate in enable() and daemon(). If we have only one instance of the event logger and taskId doesn't change in one execution. setting it once would suffice.

Reply:
discussed offline.
as we are creating a subprocess they are having different memory space